docs/guides/getting_started/samples/intro/structure.cs

```diff
@@ -65,6 +65,13 @@ private static Task Logger(LogMessage message)
         }
         Console.WriteLine($"{DateTime.Now,-19} [{message.Severity,8}] {message.Source}: {message.Message}");
         Console.ForegroundColor = cc;
+        
+        // If you get an error saying 'CompletedTask' doesn't exist,
```

Comment:
IMO you should make this comment smaller. Something like
```cs
// If the following causes an error, target .NET Framework 4.6 or higher
```
Users who are specifically targeting .NET 4.5 will likely know about Task.Delay(0)